server/tm_cmds.go

```diff
@@ -10,8 +12,24 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/cosmos-sdk/version"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+const (
+	flagEvents = "events"
```

Comment:
These flags exist in auth and in server. Could you make one of the two public and remove the other ones?
Server depends on auth, so I think best to remove these and make the one from auth public.